cmd/juju/application/refresh_test.go

```diff
@@ -693,12 +720,13 @@ func (s *RefreshSuite) TestForcedSeriesUpgrade(c *gc.C) {
 	s.charmAPIClient.CheckCall(c, 2, "SetCharm", model.GenerationMaster, application.SetCharmConfig{
 		ApplicationName: "multi-series",
 		CharmID: application.CharmID{
-			URL: charm.MustParseURL("local:trusty/multi-series-1"),
+			URL: charm.MustParseURL("local:jammy/multi-series-1"),
```

Comment:
@hmlanigan I made this change to get passing tests, but I'm unsure in the context of the test why this is the case. Does it look right to you?

Reply:
This change is okay for now. 

I incorrectly updated the setup() method not taking into account data provided by the caller. I need to go back to 2.9 and fix the test there. Then perhaps rewrite the tests for main. I think I got the test to pass without the force flag as written, so that needs to be looked at.

Reply:
https://warthogs.atlassian.net/browse/JUJU-4641